db/universities.ts

```diff
@@ -35,6 +35,17 @@ const upsertMany = async (universitiesList: any[]): Promise<any[]> => {
   return query;
 };
 
+let noUniversityNow;
+const getNoUniversityNow = async (): Promise<any> => {
+  if (!noUniversityNow) {
+    noUniversityNow = db.select('id', 'name', 'emailSSU', 'emailUniversity')
```

Comment:
As we discussed in PR #357, selecting specific fields can be removed here.
OR select only `id` and rename function to `getNoUniversityId`